core/cli/node/main.cpp

```diff
@@ -19,6 +21,28 @@ namespace fc::cli::cli_node {
           CMD("listen", Node_net_listen),
           CMD("peers", Node_net_peers),
       })},
+      {GROUP("filplus")({
+          CMD("grant-datacap", Node_client_grantDatacap),
```

Comment:
It would be nice to have command descriptions.

Reply:
I agree, we will add a description in the future